src/comparator/field_comparator.py

```diff
@@ -69,6 +69,20 @@ def compare(self):
             FindingContainer.addFinding(
                 FindingCategory.FIELD_TYPE_CHANGE, "", msg, True
             )
+        # 6. Check the oneof_index of the field.
+        oneof_original = self.field_original.HasField("oneof_index")
+        oneof_update = self.field_update.HasField("oneof_index")
+        if oneof_original != oneof_update:
+            if oneof_original:
+                msg = f"The existigng field {self.field_original.name} is moved out of One-of."
```

Comment:
nit: please fix "existing" spelling - and below

Reply:
Thanks for catching! Fixed them 👍